packages/syntax-core/src/TextField/TextField.module.css

```diff
@@ -1,6 +1,6 @@
 .textfield {
   appearance: none;
-  border: 1px solid var(--color-cambio-gray-370);
+  border: 2px solid var(--color-cambio-gray-370);
```

Comment:
looking at the slack thread, I don't think we want to change the stroke to 2px for the non-errored state